imports/plugins/core/orders/server/no-meteor/queries/ordersByAccountId.js

```diff
@@ -0,0 +1,70 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+
+/**
+ * @name ordersByAccountId
+ * @method
+ * @memberof Order/NoMeteorQueries
+ * @summary Query the Orders collection for orders made by the provided accountId and (optionally) shopIds
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} params - request parameters
+ * @param {String} params.accountId - Account ID to search orders for
+ * @param {String} params.orderStatus - Workflow status to limit search results
```

Comment:
It will be more useful and not much extra work to do `orderStatuses` array rather than just one. Return orders with any of the statuses in the array. (So in UI the filter could be check boxes rather than a single status select.)

Reply:
Good idea. How do we want to go about the `All` selection then? Just have all the statuses checked by default?

Reply:
@rymorgan See above. What are your thoughts on using a multi-select instead of a single dropdown for the Order Status filter?

I don't think it looks that great, might get confusing for the user too.

I think @aldeed's idea of passing an array is best for the server side, but on the client maybe we have pre-set arrays that we pass (at least in our starterkit, other people can do what we want), and continue to use the single select. Thoughts?

![account_profile___reaction](https://user-images.githubusercontent.com/4482263/53376984-3357cf80-3915-11e9-8dd0-c6b4bf13fc4b.png)

Reply:
Agreed that a multi-select would likely be confusing to a user. I think it adds an extra bit of complexity to the user experience for not a huge benefit to the user. And actually a detriment to the user if they are confused by the UI. I don't remember seeing this type of behavior on brands that I visit which makes me think we'd be designing something almost all storefronts would eliminate.

Reply:
I agree, too. I was thinking more of future operator UI needs.